lib/plug/conn/utils.ex

```diff
@@ -265,10 +265,13 @@ defmodule Plug.Conn.Utils do
       iex> list("empties, , are,, filtered")
       ["empties", "are", "filtered"]
 
+      iex> list("whitespace , , ,, is,optional")
+      ["whitespace", "is", "optional"]
+
   """
   @spec list(binary) :: [binary]
   def list(binary) do
-    for elem <- :binary.split(binary, ",", [:global]),
+    for elem <- :binary.split(binary, [" , ", ", ", " ,", ","], [:global]),
```

Comment:
Unfortunately this won't handle this case: "whitespace   ,". We will need to employ another approach. In `strip_spaces`, once we trim the whitespace on the front, we will need to trim it in the back. You can call a `trim_trailing(binary)` that works kinda like this:

```
defp trim_trailing(binary), do trim_trailing(binary, byte_size(binary))

defp trim_trailing(binary, pos) do
  if pos > 0 and :binary.at(binary, pos - 1) in [?\s, ?\t] do
    trim_trailing(binary, pos - 1)
  else
    binary_part(binary, 0, pos)
  end
end
```

Reply:
Yeah, thanks. That's why I added the [WIP] in front. I threw that in to pass the test case but wanted to discuss options.